src/platform/user/profile/vet360/components/base/TransactionErrorBanner.jsx

```diff
@@ -7,21 +7,17 @@ import {
   hasMVIError,
   hasMVINotFoundError,
 } from '../../util/transactions';
+import {divIcon} from 'leaflet';
```

Comment:
Oops. If VSCode has ESLint set up properly it should have complained about this

Reply:
Yes it did. I removed the import.